inst/tinytest/test_deserialization.R

```diff
@@ -150,59 +150,59 @@ expect_identical(
   target
 )
 
-target <- c("10000000000", "20000000000")
-expect_identical(
-  RcppSimdJson:::.deserialize_json(test, int64_r_type = int64_opt$string),
-  target
-)
+#target <- c("10000000000", "20000000000")
+#expect_identical(
+#  RcppSimdJson:::.deserialize_json(test, int64_r_type = int64_opt$string),
+#  target
+#)
 
-target <- structure(c(4.94065645841247e-314, 9.88131291682493e-314),
-  class = "integer64"
-)
-expect_identical(
-  RcppSimdJson:::.deserialize_json(test, int64_r_type = int64_opt$integer64),
-  target
-)
+#target <- structure(c(4.94065645841247e-314, 9.88131291682493e-314),
+#  class = "integer64"
+#
+#expect_identical(
+#  RcppSimdJson:::.deserialize_json(test, int64_r_type = int64_opt$integer64),
+#  target
+#)
 #* u64 -------------------------------------------------------------------------
-test <- "[10000000000000000000,10000000000000000001]"
-target <- c("10000000000000000000", "10000000000000000001")
-expect_identical(
-  RcppSimdJson:::.deserialize_json(test),
-  target
-)
+#test <- "[10000000000000000000,10000000000000000001]"
+#target <- c("10000000000000000000", "10000000000000000001")
+#expect_identical(
+#  RcppSimdJson:::.deserialize_json(test),
```

Comment:
As aboce, but I stop now.  A simpler may have been to renamed the test to have it skip.